README.md

```diff
@@ -1,2 +1,47 @@
-# new-website
-New IATI (about &amp; guidance) website based on Django and Wagtail CMS
+# IATI Website (pre-Alpha)
+This repository hosts the new IATI website based on Django and Wagtail CMS.  A PostgreSQL database stores the underlaying content and user data.
+
+The current scope of the project (to April 2018) focuses on the 'About IATI' and 'Publisher guidance' sections.
+
+
+## Pre-requite packages
+
+- Python3
+- PostgreSQL
+
+
+## Dev setup
+```
+# Set-up and activate virtual environment
+python3 -m venv pyenv
+source pyenv/bin/activate
+
+# Enter into project directory
+cd iati
+
+# Install requirements
+pip install -r requirements-dev.txt
+
+# Create a local PostgreSQL database (with appropriate user permissions)
+# settings/base.py expects username 'myusername' and password 'mypassword'
+# You can also use settings/local.py to set database credentials for your local instance
```

Comment:
Suggest adding a link to the relevant Django documentation about overriding base.py settings.

Reply:
The best I've found is https://github.com/wagtail/wagtaildemo/blob/master/wagtaildemo/settings/local.py.example#L1-L3 - do you think this is sufficient...?

Reply:
I can't see anything about databases in base.py now. Maybe add something super basic along the same lines as wagtail https://github.com/wagtail/wagtaildemo/blob/master/wagtaildemo/settings/base.py#L18

Reply:
Have reenabled SQLite as the default database in 091fe5e